x/gov/types/genesis.go

```diff
@@ -73,3 +74,15 @@ func ValidateGenesis(data GenesisState) error {
 
 	return nil
 }
+
+var _ types.UnpackInterfacesMessage = GenesisState{}
+
+func (data GenesisState) UnpackInterfaces(unpacker types.AnyUnpacker) error {
```

Comment:
Please add a godoc to this. State what unpacking means and why it's needed for genesis state.

Reply:
Is it satisfactory to just indicate that this implements the `UnpackInterfaceMessage.UnpackInterfaces` method? That interface and its method have detailed godocs which don't need to be duplicated everywhere

Reply:
Sure :)